libsplinter/src/oauth/builder/openid.rs

```diff
@@ -40,12 +44,26 @@ impl OpenIdOAuthClientBuilder {
         }
     }
 
+    /// Constructs a new [`OpenIdOAuthClientBuilder`] that's pre-configured with the scope for
+    /// getting refresh tokens.
+    pub fn new_azure() -> Self {
+        Self {
+            openid_discovery_url: None,
+            inner: OAuthClientBuilder::default().with_scopes(vec![AZURE_SCOPE.into()]),
```

Comment:
Does this lose the default scopes for azure?

Reply:
No, `with_scopes` appends to the existing Vec